### ISSUE TITLE
Static arrays not handled in MATCH

### ISSUE THREAD

**Issue by tomcat90** [PHPOffice/PhpSpreadsheet#1271]
This is:

```
- [X] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
MATCH with a static array should return the position of the found value based on the values submitted.

### What is the current behavior?
Returns #N/A, unless the element searched for is at the end of the array.

### What are the steps to reproduce?
Use a static array in match with the following formula. `"=MATCH(6,{4,5,6,2}, 0)";`

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

```php
<?php

use PhpOffice\PhpSpreadsheet\Spreadsheet;
use PhpOffice\PhpSpreadsheet\Writer\Xlsx;
use PhpOffice\PhpSpreadsheet\Calculation\Calculation;

$spreadsheet = new Spreadsheet();
Calculation::getInstance($spreadsheet)->disableCalculationCache();
$sheet = $spreadsheet->getActiveSheet();

//Flat Rate
$flat_rate = "=MATCH(6,{4,5,6,2}, 0)";
$sheet->getCell('A1')->setValue($flat_rate);
// echo "Flat rate\n";
echo $sheet->getCell("A1")->getValue() . "\n\n";
echo $sheet->getCell("A1")->getCalculatedValue() . "\n";
echo "\n\n";
die("HERE");
```

### Which versions of PhpSpreadsheet and PHP are affected?
1.10.1

**Comment by adjenks** [comment]
Un-stale!

**Comment by oleibman** [comment]
Fixed by PR #1332 in APril 2020.
